src/vi/goctruyentranhvui/build.gradle

```diff
@@ -0,0 +1,8 @@
+ext {
+    extName = 'Goc Truyen Tranh Vui'
+    extClass = '.GocTruyenTranhVui'
+    extVersionCode = 1
+    isNsfw = true
```

Comment:
Is there NSFW content on the website or you just copy-pasted this from another source?